doc/src/developer_guide.rst

```diff
@@ -224,6 +219,11 @@ The families of schemes follow a naming convention
     :exclude-members: _abc_cache, _abc_negative_cache,
                       _abc_negative_cache_version, _abc_registry
 
+Note that :code:`DualDenseRKPseudoStepper` includes families of
+`PseudoStepper`_ whose coefficients are read from .txt files named thus:
+
+`{scheme name}-s{stage count}-p{temporal order}-sp{optimal spatial polynomial order}.txt`
```

Comment:
We could perhaps ass :code: here to format the file name spec?

Reply:
Not sure - especially if we use it only with functions? It's now in italics so it pops up from the main text.

Reply:
OK - although not sure it is currently in italics? But fine without anyway.

Reply:
Ah ok. Not sure how it looks on the website. If I build an html using sphinx it appears in italics. Let's just remove the ` `?